ec2/instance.py

```diff
@@ -358,9 +390,14 @@ def _get_instance_attribute(attribute):
     instance_object = _get_instance_from_id(instance_id)
 
     if instance_object is None:
```

Comment:
and not external resource
....
else
raise NonRecoverableError because external resource doesn't have reservation id